packages/react-components/src/molecules/__tests__/ExternalLink.test.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import ExternalLink from '../ExternalLink';
+
+it('renders an external link', () => {
+  const { getByRole } = render(<ExternalLink href="http://example.com" />);
+  expect(getByRole('link')).toHaveAttribute('href', 'http://example.com');
+});
```

Comment:
renders the label
renders an external link icon (which should prompt us to give the icon a proper `<title>`)

Reply:
:point_up: